app/javascript/packages/document-capture/components/file-image.jsx

```diff
@@ -13,12 +14,17 @@ import useIfStillMounted from '../hooks/use-if-still-mounted';
  * @param {FileImageProps} props Props object.
  */
 function FileImage({ file, alt, className }) {
-  const [imageData, setImageData] = useState(/** @type {string?} */ (null));
+  const cache = useContext(FileBase64CacheContext);
+  const [, forceRender] = useState((prevState = 0) => 1 - prevState);
```

Comment:
I had not seen this kind of destructuring to throw away the first value, I always use an underscore for throwaway/ignored values: `[_, forceRender]`

Reply:
> I had not seen this kind of destructuring to throw away the first value, I always use an underscore for throwaway/ignored values: `[_, forceRender]`

This would trigger the [`no-unused-vars` ESLint rule](https://eslint.org/docs/rules/no-unused-vars), at least as currently configured. We could choose to make an exception pattern here using the [`varsIgnorePattern` option](https://eslint.org/docs/rules/no-unused-vars#varsignorepattern) (e.g. `^_`).

Reply:
Ah, noted, thx